lib/ansible/modules/cloud/ovirt/ovirt_templates.py

```diff
@@ -64,6 +64,29 @@
         description:
             - "Boolean indication whether to allow partial registration of a template when C(state) is registered."
         version_added: "2.4"
+    vnic_profile_mappings:
+        description:
+            - "Mapper which maps an external virtual NIC profile to one that exists in the engine when C(state) is registered.
+               vnic_profile is described by the following dictionary:
+              - C(source_network_name): The network name of the source network.
```

Comment:
The dash `-` has not good indentation, it should be at same level as the dash above.

Also You should add double quotes for every line:

- "C(source_network_name): The network name of the source network."

Same for below lines.

Reply:
done